__tests__/app.test.js

```diff
@@ -313,5 +312,86 @@ describe("Using app.js to run the database of NC-games", () => {
         });
     });
   });
+  describe("8.GET/api/reviews this can take a query!", () => {
+    test("200: response with a Array of objects with reviews", () => {
+      return request(app)
+        .get("/api/reviews")
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.reviewArray.length > 0).toBe(true);
+          expect(
+            body.reviewArray.forEach((category) => {
+              expect(category).toHaveProperty("owner", expect.any(String));
+              expect(category).toHaveProperty("title", expect.any(String));
+              expect(category).toHaveProperty("review_id", expect.any(Number));
+              expect(category).toHaveProperty("category", expect.any(String));
+              expect(category).toHaveProperty(
+                "review_img_url",
+                expect.any(String)
+              );
+              expect(category).toHaveProperty("created_at", expect.any(String));
+              expect(category).toHaveProperty("votes", expect.any(Number));
+              expect(category).toHaveProperty("designer", expect.any(String));
+              expect(category).toHaveProperty(
+                "comment_count",
+                expect.any(String)
+              );
+            })
+          );
+        });
+    });
+    test("200: response with a Array of objects with reviews but category is dexterity", () => {
+      return request(app)
+        .get("/api/reviews?category=dexterity")
+        .expect(200)
+        .then(({ body }) => {
+          expect(Array.isArray(body.reviewArray)).toBe(true);
```

Comment:
348 - we don't need this, as using body.reviewArray.forEach on line 351 implies that it is an array